expg.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
it would've been nice to see a deployed version of your editor... otherwise it is very simple and easy to follow! nice!!

Reply:
thanks! Ive just uploaded an img of it in the browser :)